code/modules/mob/living/basic/space_fauna/spider/giant_spider/giant_spiders.dm

```diff
@@ -246,6 +247,97 @@
 		return FALSE
 	return TRUE
 
+/**
+ * ### Spider Tank
+ * A subtype of the giant spider which is similar on every single way,
+ * This spider is only slightly slower than a human.
+ */
+/mob/living/basic/spider/giant/tank
+	name = "tank spider"
+	desc = "Furry and Purple with a white top, it makes you shudder to look at it. This one has bright yellow eyes."
+	icon_state = "tank"
+	icon_living = "tank"
+	icon_dead = "tank_dead"
+	maxHealth = 500
+	health = 500
+	damage_coeff = list(BRUTE = 1, BURN = 1, TOX = 1, CLONE = 1, STAMINA = 1, OXY = 1)
+	melee_damage_lower = 5
+	melee_damage_upper = 5
+	obj_damage = 15
+	speed = 5
+	player_speed_modifier = -4
+	menu_description = "Extremly tanky with very little damage able to self heal and lay reflective silk screens."
+
+/mob/living/basic/spider/giant/tank/Initialize(mapload)
+	. = ..()
+	var/datum/action/cooldown/mob_cooldown/lay_web/web_reflector/reflector_web = new(src)
+	reflector_web.Grant(src)
+
+	var/datum/action/cooldown/mob_cooldown/lay_web/web_passage/passage_web = new(src)
+	passage_web.Grant(src)
+
+	AddElement(/datum/element/web_walker, /datum/movespeed_modifier/below_average_web)
+
+	AddComponent(/datum/component/healing_touch,\
```

Comment:
I'll be honest I find the "click yourself to heal" mechanics to be a little unintuitive. Would the regenerator component as a passive replacement work alright instead?

Reply:
The one main difference is healing touching yourself allows some spider types to survive in damaging environments in order to set up webbing for everyone else.  If its replaced with regenerator, this won't be possible.